app/src/main/java/ch/hikemate/app/MainActivity.kt

```diff
@@ -54,45 +54,52 @@ fun HikeMateApp() {
 
   val authViewModel = AuthViewModel(FirebaseAuthRepository())
 
-  NavHost(navController = navController, startDestination = TopLevelDestinations.AUTH.route) {
-    navigation(
-        startDestination = Screen.AUTH,
-        route = Route.AUTH,
-    ) {
-      composable(Screen.AUTH) { SignInScreen(navigationActions, authViewModel) }
-      composable(Screen.SIGN_IN_WITH_EMAIL) {
-        SignInWithEmailScreen(navigationActions, authViewModel)
-      }
-      composable(Screen.CREATE_ACCOUNT) { CreateAccountScreen(navigationActions, authViewModel) }
-    }
+  val isUserLoggedIn = authViewModel.isUserLoggedIn()
```

Comment:
Question: I am confused about what is going on here, did the code actually change? I think its just GitHub convoluting the changes to look more than they actually are, right?

Reply:
Yes, as I said in the PR description, the whole code is in change because it changes the formatting. I only changed the code to use `isUserLoggesIn` to decide if we display `AUTH` or `MAP`. The difference is that we now do that in `MainActivity` instead of `SignInScreen` so that the Sign in screen will no longer show up.

Reply:
I see, thanks for clarifying.